aries-site/src/pages/foundation/date-and-time.mdx

```diff
@@ -0,0 +1,259 @@
+| [Absolute time](#absolute-time) | To provide the precise date and/or time of an event.                            | September 15, 2023; 3/27/2023 | The granularity and formatting depends on the context. See absolute time guidance. |
+| [Relative time](#relative-time) | To display how long ago something occurred or in how long something will occur. | in 5 days, 3 days ago         | --                                                                                 |
+
+Timestamps should always be stored in UTC, the universal standard. These UTC timestamps will then get converted to local time, if necessary, by the UI for display.
+
+## Absolute time
+
+Absolute time stamps display the precise date and time an action has occurred or will occur.
+
+May be presented as:
+
+- Date
+- Date and time
+- Including seconds (and fractional seconds)
+- Including timezone
+
+In most cases, absolute time should be presented according to the user's [locale](#locale-formatting) and [timezone](#timezones), indicating the timezone in the output.
+
+Absolute time can be presented in a variety of formats:
+
+export const date = new Date('2023-08-12');
+
+| Formats                        | Example                                                                                                           | When to use                                                                                                                                                                                                                                                                            | Code snippet (where date is a Javascript Date object)                                                                            |
+| ------------------------------ | ----------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------- |
+| Long written format            | {Intl.DateTimeFormat('en-US', { weekday: 'short', month: 'long', day: 'numeric', year: 'numeric' }).format(date)} | When space is available and displaying the day of week is beneficial to the user.                                                                                                                                                                                                      | <Text>{`Intl.DateTimeFormat('en-US', { weekday: 'short',  day: 'numeric',month: 'long', year: 'numeric' }).format(date)`}</Text> |
```

Comment:
I'll give it a try to format it like that. In general, do you think having the code snippet is valuable?

Reply:
Wondering if instead of hard coding to `en-US`, we should do 
```
const locale = navigator.language;

....

`Intl.DateTimeFormat(locale, { weekday: 'short'`Intl.DateTimeFormat('en-US', { weekday: 'short' ...`
```

OR, I think using `undefined` uses the browser's default locale.

```
`Intl.DateTimeFormat(undefined, { weekday: 'short'`Intl.DateTimeFormat('en-US', { weekday: 'short' ...`
```

Reply:
Done.